FilterReview/index.html

```diff
@@ -94,14 +131,22 @@ <h1 style="text-align:center"><a href="" style="color: #000000; text-decoration:
 </table>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Flight Data</h2>
+    <h2 style="text-align:center">Flight Data
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Zoom into a section of the flight to change the Analysis time then click "Calculate". Selecting a hover section when the noise should be constant will give the clearest results on the IMU spectrum',
+        data-tippy-maxWidth='750px'/>
+    </h2>
 </td></tr></table>
 
 <div id="FlightData" style="width:1200px;height:450px"></div>
 
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">IMU Spectrum</h2>
+    <h2 style="text-align:center">IMU Spectrum
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Spectrums are shown for each Gyro. Each axis can be shown or hidden, before the filters, after any enabled filtering, and/or an estimate after any proposed filtering made by changing the logs values in the "Filter Configuration" panel. Double Gyro headings click to show/hide all.',
```

Comment:
```suggestion
        data-tippy-content='Spectrums are shown for each Gyro. Each axis can be shown or hidden, before the filters, after any enabled filtering, and/or an estimate after any proposed filtering made by changing the log's values in the "Filter Configuration" panel. Double click Gyro headings to show/hide all.',
```

Reply:
The I had to use Unicode to put in the apostrophe without messing up the sting since it has double quotes in it already.